app/src/ui/lib/popover.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react'
+import FocusTrap from 'focus-trap-react'
+import { Options as FocusTrapOptions } from 'focus-trap'
+
+/**
+ * Position of the caret relative to the pop up. It's composed by 2 dimensions:
+ * - The first one is the edge on which the caret will rest.
+ * - The second one is the alignment of the caret within that edge.
+ *
+ * Example: TopRight means the caret will be in the top edge, on its right side.
```

Comment:
Very helpful documentation!

Reply:
Think it might be useful to add documentation explaining that this will add a class onto the popupover and to add more position options requires adding new classes to the popover.scss?

Reply:
I considered that, yeah, but I didn't add it to this specific comment block because that's public and the class thing is an implementation detail. But I think it's worth commenting it somewhere/somehow regardless!